src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
```

Comment:
The `crossorigin` attribute is missing here. When using `preconnect`, it's good practice to include `crossorigin` if you're connecting to a domain that will serve CORS-enabled resources.